packages/fast-tooling-react/src/form/form-dictionary.tsx

```diff
@@ -42,20 +73,22 @@ class FormDictionary extends React.Component<
             formDictionary_controlRegion,
         }: FormDictionaryClassNameContract = this.props.managedClasses;
 
-        return (
-            <div className={formDictionary_controlRegion}>
-                <div className={formDictionary_control}>
-                    <label className={formDictionary_controlLabel}>
-                        {this.props.label}
-                    </label>
+        if (typeof this.props.additionalProperties === "object") {
```

Comment:
You might want to use lodash's `isPlainObject` so data like `null` and `Date` don't pass this condition.